index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/my_portfolio.css">
+</head>
+<body>
+    <header id="header" class="header"> 
+        <div class="logo">John Doe</div>
```

Comment:
- [ ] Hi, well done on including the logo. Although this logo should have been implemented using an `a` tag. If you look at most sites clicking on the logo usually links back to the homepage. this will make your website much better 🤗
```suggestion
        <a href="/" class="logo">John Doe</a>
```